pubspec.yaml

```diff
@@ -10,7 +10,7 @@ authors:
   - Trent Grover <trent.grover@workiva.com>
 homepage: https://github.com/Workiva/w_flux
 dependencies:
-  react: ">=0.5.0 <2.0.0"
+  react: ">=0.5.0 <3.0.0"
```

Comment:
and I checked: it's not possible in dart to include the same lib in both deps and dev_deps